src/geometry/Point.js

```diff
@@ -100,73 +95,71 @@ Point.prototype = {
 	// defined by `scale`.
 	scaleBy(point) {
 		return new Point(this.x * point.x, this.y * point.y);
-	},
+	}
 
 	// @method unscaleBy(scale: Point): Point
 	// Inverse of `scaleBy`. Divide each coordinate of the current point by
 	// each coordinate of `scale`.
 	unscaleBy(point) {
 		return new Point(this.x / point.x, this.y / point.y);
-	},
+	}
 
-	// @method round(): Point
 	// Returns a copy of the current point with rounded coordinates.
 	round() {
 		return this.clone()._round();
-	},
+	}
 
 	_round() {
 		this.x = Math.round(this.x);
 		this.y = Math.round(this.y);
 		return this;
-	},
+	}
 
 	// @method floor(): Point
 	// Returns a copy of the current point with floored coordinates (rounded down).
 	floor() {
 		return this.clone()._floor();
-	},
+	}
 
 	_floor() {
 		this.x = Math.floor(this.x);
 		this.y = Math.floor(this.y);
 		return this;
-	},
+	}
 
 	// @method ceil(): Point
 	// Returns a copy of the current point with ceiled coordinates (rounded up).
 	ceil() {
 		return this.clone()._ceil();
-	},
+	}
 
 	_ceil() {
 		this.x = Math.ceil(this.x);
 		this.y = Math.ceil(this.y);
 		return this;
-	},
+	}
 
-	// @method trunc(): Point
 	// Returns a copy of the current point with truncated coordinates (rounded towards zero).
 	trunc() {
 		return this.clone()._trunc();
-	},
+	}
 
 	_trunc() {
-		this.x = trunc(this.x);
-		this.y = trunc(this.y);
+		this.x = Math.trunc(this.x);
+		this.y = Math.trunc(this.y);

```

Comment:
`Math.trunc` is widely supported according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/trunc.